CHANGES.md

```diff
@@ -2,6 +2,13 @@
 
 ## not yet released
 
+## 2.3.1
+
+- `MANTA-4799 add branch protections on "mantav1" branches of the relevant manta repos`
+  This updates the `jr gh` `check` and `set-branch-protection` subcommands so
+  that they also operate on the `mantav1` branch of the given repository or
+  repositories, if that branch exists.
```

Comment:
Is it if the branch exists? or if the repo has a "mantav1:true" label in repo metadata?

XXX note to self to check

Reply:
I see from the code it *is* if the branch exists, so all good here.